src/compiler/compile/nodes/Element.ts

```diff
@@ -710,7 +710,10 @@ export default class Element extends Node {
 		}
 
 		if (this.name === 'video') {
-			if (attribute_map.has('muted')) {
+			const aria_hidden_attribute = attribute_map.get('aria-hidden');
+			const aria_hidden_exist = aria_hidden_attribute && aria_hidden_attribute.get_static_value();
+
+			if (attribute_map.has('muted') || aria_hidden_exist) {
```

Comment:
Do we need to omit the warnings only if the value is `true`?

```suggestion
			if (attribute_map.has('muted') || aria_hidden_exist === 'true')
```

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#values